packages/@okta/vuepress-site/docs/reference/api/authorization-servers/index.md

```diff
@@ -1501,16 +1497,16 @@ Example from a Policy Object
 }
 ```
```

Comment:
This highlighting tells me that there's an invalid character somewhere around here.

Reply:
The JSON is invalid. I'll push a fix.